usmqe_tests/rpm/test_rpm.py

```diff
@@ -74,13 +74,13 @@ def test_rpmlint(rpm_package):
     cp = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     LOGGER.debug("STDOUT: %s", cp.stdout)
     LOGGER.debug("STDERR: %s", cp.stderr)
+    LOGGER.debug("RCODE: %s", cp.returncode)
     # when the check fails, report the error in readable way
```

Comment:
Could you update the comment here as well: `report every line on stdout as an error in readable way`